### PR TITLE
docs: revamp design

### LINKED ISSUE #181: New website: Revamp Docs' Design
## Requirements

- [ ] Should look like the new [design](https://www.figma.com/file/Ri5LvLDhDLhXVAAAU7pHh2/Monika-Website?node-id=149%3A386).

### PR DESCRIPTION
**Pull request by samsunoor**
This PR resolves #181 
## How to test
1. Open file `docs/src/libs/docs/md-loader.js`
2. Change line 10 to `const layout = data.layout || 'DocsNtmy'`
3. Move to terminal
4. `cd ./docs`
5. `npm run dev`
6. Visit any documentation page, e.g., `localhost:3000/overview`
7. Need to test mobile version visuals, too

### CODE REVIEW

File: docs/src/components/Button.js [hunk 1]
```diff
@@ -24,30 +24,29 @@
...
 
+  const isRounded = props.rounded !== false
+
   return (
     <button
-      className={`px-4 py-2 bg-gradient-to-r from-purple-monika to-aqua-monika rounded-full font-sans text-white ${
+      className={`px-4 py-2 bg-gradient-to-r from-purple-monika to-aqua-monika font-sans text-white ${
         props.className ? props.className : ''
-      }`}
+      } ${isRounded ? 'rounded-full' : 'rounded-md'}`}
```
Comment (kevinhermawan) [inline docs/src/components/Button.js:36]: You don't need to declare constant, just use `props.rounded`.

File: docs/src/components/Button.js [hunk 2]
```diff
@@ -24,30 +24,29 @@
...
 
 function ButtonOutlined(props) {
+  const isRounded = props.rounded !== false
   return (
     <button
-      className={`px-4 py-2 rounded-full border-2 border-purple-monika text-purple-monika font-sans ${
+      className={`px-4 py-2 border-2 border-purple-monika text-purple-monika font-sans ${
         props.className ? props.className : ''
-      }`}
+      } ${isRounded ? 'rounded-full' : 'rounded-md'}`}
```
Comment (kevinhermawan) [inline docs/src/components/Button.js:49]: This too

**Review by kevinhermawan — APPROVED** [review]

**Review by sapiderman — APPROVED** [review]
works!

File: docs/src/components/LayoutDocsNtmy.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          title={props.meta.title || title}
+          description={props.meta.description}
+        />
+        <div className="block">
+          <>
+            <div className="container mx-auto pb-12 pt-6 content">
+              <div className="flex relative">
+                {!isMobile && (
+                  <Sidebar fixed>
+                    <SidebarRoutes routes={routes} />
```
Comment (raosan) [inline docs/src/components/LayoutDocsNtmy.js:84]: there is a warning in developer console saying there are duplicate keys
Screen Shot 2021-06-24 at 16 59 19

**Review by raosan — CHANGES_REQUESTED** [review]
as all component has been implemented, we can replace the original component and using the ntmy component so we can check without changing md-loader.js

**Review by raosan — APPROVED** [review]

**Pull request merged**